substrate/frame/delegated-staking/src/lib.rs

```diff
@@ -493,13 +488,8 @@ impl<T: Config> Pallet<T> {
 
 	/// Registers a new agent in the system.
 	fn do_register_agent(who: &T::AccountId, reward_account: &T::AccountId) {
+		// TODO: Consider taking a deposit for being an agent.
```

Comment:
I'd suggest opening a follow up issue and include the link here (or add one if it already exists).

Reply:
https://github.com/paritytech/polkadot-sdk/issues/5368